src/inventory/gilded_rose.js

```diff
@@ -22,49 +22,80 @@ const items = [
 updateQuality(items);
 */
 export function updateQuality(items) {
+
   for (var i = 0; i < items.length; i++) {
-    if (items[i].name != 'Aged Brie' && items[i].name != 'Backstage passes to a TAFKAL80ETC concert') {
-      if (items[i].quality > 0) {
-        if (items[i].name != 'Sulfuras, Hand of Ragnaros') {
-          items[i].quality = items[i].quality - 1
+
+   //------VARIABLES-----//
+   
+    //create item variables to simplify & improve readability, decrease repetition, 
+    const item = items[i]
+    const agedBrie = ['Aged Brie'];
+    const backstagePass = ['Backstage passes to a TAFKAL80ETC concert'];
+    const constantQuality = ['Sulfuras, Hand of Ragnaros'];
+
+    if (!agedBrie.includes(items[i].name)&& !backstagePass.includes(items[i].name)) {
+      if (item.quality > 0) {
+        if (!constantQuality.includes(items[i].name)){
+          item.quality = item.quality - 1
         }
       }
     } else {
-      if (items[i].quality < 50) {
-        items[i].quality = items[i].quality + 1
-        if (items[i].name == 'Backstage passes to a TAFKAL80ETC concert') {
-          if (items[i].sell_in < 11) {
-            if (items[i].quality < 50) {
-              items[i].quality = items[i].quality + 1
+      if (item.quality < 50) {
+        item.quality = item.quality + 1
+        if (backstagePass.includes(items[i].name)) {
+          if (item.sell_in < 11) {
+            if (item.quality < 50) {
+              item.quality = item.quality + 1
             }
           }
-          if (items[i].sell_in < 6) {
-            if (items[i].quality < 50) {
-              items[i].quality = items[i].quality + 1
+          if (item.sell_in < 6) {
+            if (item.quality < 50) {
+              item.quality = item.quality + 1
             }
           }
         }
       }
     }
-    if (items[i].name != 'Sulfuras, Hand of Ragnaros') {
-      items[i].sell_in = items[i].sell_in - 1;
+    if (!constantQuality.includes(items[i].name)) {
+      item.sell_in = item.sell_in - 1;
     }
-    if (items[i].sell_in < 0) {
-      if (items[i].name != 'Aged Brie') {
-        if (items[i].name != 'Backstage passes to a TAFKAL80ETC concert') {
-          if (items[i].quality > 0) {
-            if (items[i].name != 'Sulfuras, Hand of Ragnaros') {
-              items[i].quality = items[i].quality - 1
-            }
+
+    //where items expire
+    //combine these into quality changer--takes in a + or - number and you can get BOTH increase or decrease in quality
+    qualityUpdater = (item, change_Amount) =>{
```

Comment:
This should include a declaration.
```suggestion
    const qualityUpdater = (item, change_Amount) =>{
```